From b6759112d4de0fb1d9902e6b3e5eb946249e9f83 Mon Sep 17 00:00:00 2001
From: J. Bruce Fields <bfields@redhat.com>
Date: Mon, 23 Jun 2014 21:50:23 -0400
Subject: [fs] nfsd: cleanup unneeded including linux/export.h

Message-id: <1403560314-32525-82-git-send-email-bfields@redhat.com>
Patchwork-id: 84517
O-Subject: [Patch RHEL7 081/172] NFSD: cleanup unneeded including linux/export.h
Bugzilla: 1111719
RH-Acked-by: Steve Dickson <SteveD@redhat.com>

From: Kinglong Mee <kinglongmee@gmail.com>

commit 4ac7249ea5a0ceef9f8269f63f33cc873c3fac61 have remove all EXPORT_SYMBOL,
linux/export.h is not needed, just clean it.

Signed-off-by: Kinglong Mee <kinglongmee@gmail.com>
Signed-off-by: J. Bruce Fields <bfields@redhat.com>
Upstream: ecc7455d8eb1860f5aa6b9ad82a9a81f93eb11d1
Bugzilla: 1111719
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfsd/nfs4acl.c b/fs/nfsd/nfs4acl.c
index bb220d9..1eec850 100644
--- a/fs/nfsd/nfs4acl.c
+++ b/fs/nfsd/nfs4acl.c
@@ -36,7 +36,6 @@
 
 #include <linux/slab.h>
 #include <linux/nfs_fs.h>
-#include <linux/export.h>
 #include "nfsd.h"
 #include "acl.h"
 
-- 
1.7.1